lib/addMethod/index.js

```diff
@@ -1,8 +1,47 @@
+const _ = require('lodash');
+
+const addMethodREST = require('./addMethodREST');
+const addMethodSOAP = require('./addMethodSOAP');
+
+const INVALID_TYPE_ERROR_MESSAGE = '`type` must be strings \'REST\' or \'SOAP\'';
```

Comment:
instead of escaping the single quotes, could use " for the surrounding quotes instead